libs/postgres_ffi/wal_craft/src/xlog_utils_test.rs

```diff
@@ -0,0 +1,222 @@
+//! Tests for postgres_ffi xlog_utils module. Put it here to break cyclic dependency.
+
+#[cfg(test)]
+mod tests {
```

Comment:
I wonder if this could just be gotten rid of because the whole file is this module? use `#[cfg(test)]` from the xlog_utils_test macro?